Sources/PackageLoading/JSONSerialization.swift

```diff
@@ -12,61 +12,5 @@ import Basic
 import PackageDescription
```

Comment:
Let's just kill this file, and make this an extension inside `PackageModel` which feels more conceptually appropriate.